src/main/resources/citizen/templates/document/defendantResponseReceipt.html

```diff
@@ -352,6 +352,336 @@
+          </td>
+        </tr>
+        {% endif %}
+      {% endfor %}
+      <tr>
+        <td colspan="2" class="keep-white-space"><strong>Maintenance:</strong></td>
+      </tr>
+      <tr class="no-border">
+        <td><strong>Number of children:</strong></td>
+        <td>
+          {{ dependant.numberOfMaintainedChildren }}
+        </td>
+      </tr>
+      <tr>
+        <td colspan="2" class="keep-white-space"><strong>Financial support:</strong></td>
+      </tr>
+      <tr class="no-border">
+        <td><strong>Do they support anyone else financially?</strong></td>
+        <td>
+        {% if employment is defined and employment is not empty %}
+          Yes
+        {% else %}
+          No
+        {% endif %}
+        </td>
+      </tr>
+      <tr class="no-border">
+        <td><strong>Number of people:</strong></td>
+        <td>
+          {{ otherDependants.numberOfPeople }}
+        </td>
+      </tr>
+      <tr class="no-border">
+        <td><strong>Give details:</strong></td>
+        <td>
+          {{ otherDependants.details }}
+        </td>
+      </tr>
+      <tr>
+        <td colspan="2" class="keep-white-space"><strong>Employment details:</strong></td>
+      </tr>
+      <tr class="no-border">
+        <td><strong>Are they currently working?</strong></td>
+        <td>
+        {% if employment is defined and employment is not empty %}
+          Yes
+        {% else %}
+          No
+        {% endif %}
+        </td>
+      </tr>
+      <tr class="no-border">
+        <td><strong>Type of job:</strong></td>
+        {% if jobType is defined %}
+        <td>
+          {{ jobType }}
+        </td>
+        {% endif %}
+      </tr>
+      <tr>
+        <td colspan="2" class="keep-white-space"><strong>Who employs them?</strong></td>
+      </tr>
+      <tr class="no-border">
+        {% for employer in employment.employers %}
+        <td><strong>Employer name:</strong></td>
+        <td>
+          {{ employer.name }}
+        </td>
+      </tr>
+      <tr class="no-border">
+        <td><strong>Job title:</strong></td>
+        <td>
+          {{ employer.jobTitle }}
+        </td>
+      </tr>
+      {% endfor %}
+
+      {% if selfEmployment is defined %}
+      <tr>
+        <td colspan="2" class="keep-white-space"><strong>Self-employment details:</strong></td>
+      </tr>
+      <tr class="no-border">
+        <td><strong>Job title:</strong></td>
+        <td>
+          {{ selfEmployment.jobTitle }}
+        </td>
+      </tr>
+      <tr class="no-border">
+        <td><strong>Annual turnover:</strong></td>
+        <td>
+          {{ selfEmployment.annualTurnover }}
+        </td>
+      </tr>
+      <tr class="no-border">
+        <td><strong>Are they behind on tax payments?</strong></td>
+        <td>
+        {% if onTaxPayments is defined %}
+          Yes
+        {% else %}
+          No
+        {% endif %}
+        </td>
+      </tr>
+      <tr class="no-border">
```

Comment:
I guess this should be conditional if somebody said `No` on `on behind tax payments` page.

Reply:
done